Source/Resources/en.lproj/Localizable.strings

```diff
@@ -1,4 +1,16 @@
-
+"missingBillingFormFullName" = "Missing Name";
```

Comment:
@ehab-al-cko Did we check with @patrick-hoban-cko to support other languages ? . we can create a separate ticket to track this work.

Reply:
@patrick-hoban-cko please share with rest of translations